packages/plugin-loansresearch-ui/src/components/detail/LoansResearchDetails.tsx

```
@@ -0,0 +1,34 @@
+
+  const breadcrumb = [
+    { title: __('Loans Research'), link: '/loansresearch' },
+    { title },
+  ];
+
+  return (
+    <Wrapper
+      header={<Wrapper.Header title={title} breadcrumb={breadcrumb} />}
+      leftSidebar={<LeftSidebar {...props} />}
+      rightSidebar={<RightSidebar loansResearch={loansResearch} />}
+      content={<></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)